findmy/accessory.py

```diff
@@ -236,6 +307,21 @@ def __init__(
 
         self._iter_ind = 0
 
+    @property
+    def master_key(self) -> bytes:
```

Comment:
adding these properties isn't strictly needed in this PR, but it seemed good to have.